loadtest/user/userentity/helper_test.go

```diff
@@ -6,29 +6,39 @@ package userentity
 import (
 	"testing"
 
-	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/defaults"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest/store/memstore"
 
 	"github.com/stretchr/testify/require"
 )
 
+type config struct {
```

Comment:
These changes are not clear to me. Are they related to this PR?

Reply:
They were, the first implementation introduced a cyclic dependency here. Didn't reverted because it was a non-essential dependency. 1/5 on this anyway, can revert back as well.

Reply:
Fine by me.